tests/datasource/fluent/test_spark_s3_datasource.py

```diff
@@ -29,34 +26,11 @@
 logger = logging.getLogger(__file__)
 
 
-@pytest.fixture()
-def aws_region_name() -> str:
-    return "us-east-1"
-
-
 @pytest.fixture()
 def aws_s3_bucket_name() -> str:
     return "test_bucket"
 
 
-@pytest.fixture(scope="function")
-def aws_credentials() -> None:
-    """Mocked AWS Credentials for moto."""
-    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
-    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
-    os.environ["AWS_SECURITY_TOKEN"] = "testing"
-    os.environ["AWS_SESSION_TOKEN"] = "testing"
-    os.environ["AWS_DEFAULT_REGION"] = "testing"
-
-
-@pytest.mark.skipif(not aws.boto3)
```

Comment:
These marks were causing the CI failure.
I've moved these to `conftest.py` and removed the marks.